tests/test_plotting.py

```diff
@@ -128,5 +184,15 @@ def test_legend(self):
         self.assertEqual(labels, expected)
 
 
+def _check_colors(collection, expected_colors):
+
+    import matplotlib.colors as colors
+    conv = colors.colorConverter
+
+    for patch, color in zip(collection, expected_colors):
+        result = patch.get_color()
+        assert conv.to_rgba(result) == conv.to_rgba(color)
```

Comment:
Following the documentation `to_rgba` always returns a tuple, so I don't think a numpy function would be needed therefore. But something else than a plain assert to deal with eg possible float rounding errors is indeed maybe a good idea.

But in this case, this is not the problem, as the difference is quite big: ``At index 0 diff: 0.89411765336990356 != 0.64705884456634521``. So will have to investigate where the error is coming from.